configure.ac

```diff
@@ -1,313 +1,127 @@
-    AC_MSG_RESULT([Found scipoptsuite in $scipdir])
-    scipincludedir="$scipdir"/src
-    scipoptsuitelibdir="$scipoptsuitedir"/lib  # this is where libscipopt is built
-    sciplibdir="$scipdir"/lib		       # this is where libscip is built
-    AS_IF([test -f "$scipoptsuitelibdir"/libscipopt.a ],
-        [AC_MSG_RESULT([ Found short link $a ]) ],
-        [# short link missing (not installed by 3.2.0, exists in 3.2.1)
-	 for a in "$scipoptsuitelibdir"/libscipopt-*.*.a ; do
-	    AS_IF([test -f $a ],
-	       [AC_MSG_RESULT([Found library $a])
-	        AS_IF([test x$scipsuffix != x ],
-		   [AC_MSG_ERROR([Directory $scipoptsuitelibdir contains more than one libscipopt*.a library, please create a link to libscipopt.a ])])
-		scipsuffix=`echo $a | sed 's|.*/libscipopt\(@<:@^/@:>@*\)\.a|\1|'`
-	       ])
-         done
-	 AS_IF([test x$scipsuffix = x],
-	     [AC_MSG_ERROR([Directory $scipoptsuitelibdir does not contain a libscipopt*.a library])])
-	])
-    LDFLAGS="-L$scipoptsuitelibdir $LDFLAGS"
-    CPPFLAGS="-I $scipincludedir $CPPFLAGS"
-   ])
-AC_ARG_ENABLE([scip],
-    [AS_HELP_STRING([--enable-scip@<:@=ARG@:>@],
-      [enable cone decomposition with SCIP @<:@default=check@:>@])],
-    [:],
-    [enable_scip=check])
-AS_IF([test "x$enable_scip" != xno],
-    [AC_MSG_CHECKING([whether SCIP headers and library are available])
-     dnl SCIP can be installed in many different ways.  We try several.
-     dnl There's libscipopt and there's libscip (which depends on
-     dnl liblpispx and libsoplex or another LP solver library; but these
-     dnl dependencies may not be recorded).
-     LIBS_SAVED="$LIBS"
-     for SCIP_LIBS in \
-	 "-lscip" \
-	 "-lscipopt$scipsuffix -lgmp -lz -lreadline" \
-	 "-lscipopt$scipsuffix -lz -lreadline" \
-	 "-lscipopt$scipsuffix -lgmp -lz" \
-	 "-lscipopt$scipsuffix -lz" \
-	 "-lscipopt$scipsuffix -lgmp -lreadline" \
-	 "-lscipopt$scipsuffix -lreadline" \
-	 "-lscipopt$scipsuffix -lgmp" \
-	 "-lscipopt" \
-	 "-lscip -lnlpi.cppad -llpispx -lsoplex -lgmp -lz -lreadline" \
-	 ; do
-	 LIBS="$LIBS_SAVED $SCIP_LIBS"
-	 dnl We use AC_RUN_IF_ELSE because on Mac OS X, running
-	 dnl the program is the only way to know whether we got
-	 dnl all dylibs working.
-	 AC_RUN_IFELSE(
-	   [AC_LANG_PROGRAM([[#include <scip/scip.h>
-				     ]],
-			[[ SCIP* scip = NULL;
-			   SCIPcreate(& scip);
-			   return 0;
-			 ]])],
-	   [have_scip=yes
-	    AC_MSG_RESULT([yes, using $SCIP_LIBS])
-	    break
-	   ],
-	   [have_scip=no])
-     done
-     LIBS="$LIBS_SAVED"
-     if test $have_scip != yes ; then
-	AC_MSG_RESULT($have_scip)
-     fi
+# We can use SCIP (Solving Constraint Integer Programs) for cone decomposition
+AC_ARG_WITH([scip], AS_HELP_STRING([--with-scip=FLAGS], [Use SCIP for cone decomposition; the FLAGS specify the linker flags necessary, e.g., -lscip. Typically, a subset of -lscip -lz -lreadline -lscipopt -lnlpi.cppad -llpispx -lsoplex or similar is going to make this work.]))
```

Comment:
So you both removed the convenience code which made it a bit easier to link against SCIP, *and* also the INSTALL instructions related to SCIP. That strikes me as potentially problematic, though I never tried to use SCIP with Normaliz myself, so perhaps there is a reason why all this code and the instructions should go -- but then IMHO the pull request should explicitly explain them.

I also wonder: Did you test the whole thing with SCIP at all?

Reply:
No. As I wrote, I did not test this. At the moment, this was just about pointing out how a `configure.ac` should roughly look like.

Reply:
OK. But as @w-bruns wrote that SCIP support perhaps can be removed completely (did I get that right), that may be the better option -- i.e., let's first remove SCIP support completely in a separate PR, that then also makes reviewing easier :-). I can look into doing that (but would be best if @w-bruns first confirms that I did not misunderstand :-).